create_bash_script.py

```diff
@@ -1,54 +1,70 @@
+import os
 import sys
 
-with open('docker/Dockerfile', 'r') as f:
+
+with open("docker/Dockerfile", "r") as f:
     dockerfile_lines = f.readlines()
 
-bash_instructions = ['#!/bin/bash']
-docker_command = ''
+claasp_directory = os.path.abspath(".")
+bash_instructions = ["#!/bin/bash"]
 environment_variables = []
+docker_command = ""
+
 for line in dockerfile_lines:
     line = line.strip()
 
-    if line.startswith("FROM claasp-base AS claasp-lib"):
```

Comment:
This section needs to stay as it was as this docker stage is not necessary for local installation. The reading of the Dockerfile should stop when this sentence is found.